src/CMakeLists.txt

```diff
@@ -571,16 +587,21 @@ function(add_doom_tiny SUFFIX RENDER_LIB)
         NO_IERROR=1
     )
     if (PICO_ON_DEVICE)
+        # target_link_libraries(doom_tiny${SUFFIX} PRIVATE hardware_spi)
+        # TODO use defs or targets for these?
+        target_link_libraries(doom_tiny${SUFFIX} PRIVATE pico_st7789 hardware_spi)
```

Comment:
these could get broken out into the different screen targets - and they may, in the future. We have #if guards around their header inclusion, so they should be factored out by the compiler anyways